docs/faq.md

```diff
@@ -10,55 +10,46 @@ hide:
+## <span class="custom-heading" data-id="2">How does ExpressLRS compare to other systems?</span>
 
-??? faq "Is it better than the commercial solutions?"
-    It is faster than most links out there with a latency of 6.5ms (at 200hz). At 250hz 100mW 2.4GHz it is capable of ranges 30km+.
-## <span class="custom-heading" data-id="3">Which OpenTX version works with ExpressLRS</span>
+??? faq "How does ExpressLRS compare to other systems?"
+    ExpressLRS is capable of extremely high packet rates (up to 1000 Hz) and extreme sensitivity (ranges well over 100+ km have been achieved with 2.4 GHz hardware), making it extremely competitive with any commercially available system. And since ExpressLRS is open-source, it costs much less than other systems too!
+## <span class="custom-heading" data-id="3">Which OpenTX/EdgeTX version do I need and why should I update?</span>
 
-??? faq "Which OpenTX version works with ExpressLRS"
-    [OpenTX 2.3.12](https://www.open-tx.org/downloads.html#Releases23-ref) or newer should work just fine. No need to use OpenTX Nightly, unless you have requirements not present in the Stable versions.
+??? faq "Which OpenTX/EdgeTX version do I need and why should I update?"
+    ExpressLRS requires [OpenTX 2.3.12](https://www.open-tx.org/downloads.html#Releases23-ref) or newer, as it depends on CRSFshot (a.k.a mixer sync) to work properly. However, as OpenTX is no longer being maintained, it is strongly recommended that you update to [EdgeTX](https://github.com/EdgeTX/edgetx) in order to take advantage of the latest performance improvements and bug fixes. 
+	
+	Click [here](https://www.expresslrs.org/quick-start/transmitters/tx-prep.md) to read more about radio firmware requirements and setup.
 
-    [EdgeTX 2.4.0](https://github.com/EdgeTX/edgetx) or newer should work too.
-## <span class="custom-heading" data-id="4">Why do I need to update OpenTX?</span>
+## <span class="custom-heading" data-id="4">How do I flash/update my receiver/module?</span>
 
-??? faq "Why do I need to update OpenTX?"
-    ExpressLRS needs your radio to support crsfshot (a.k.a. Mixersync) to work properly. This will give you the lowest possible latency and optimal consistency of the RC link. When your radio does not have crsfshot working, this often shows in your ExpressLRS Lua script. The Lua script top bar will show inconsistent numbers like 0:63 or is stuck at 0:250 at every packet rate rate you select.
+??? faq "How do I flash/update my receiver/module?"
+    See [Getting Started](https://www.expresslrs.org/quick-start/getting-started.md) for a Quick Setup guide and detailed instructions on flashing, features, hardware, and troubleshooting.
```

Comment:
should it be relative URL instead of the absolute one?

Reply:
Changed